tests/modules/commerce_test/commerce_test.module

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Module file for commerce_test.
+ */
+
+use Drupal\commerce_price\Price;
+
+/**
+ * Implements hook_theme().
+ */
+function commerce_test_theme($existing, $type, $theme, $path) {
+  return [
+    'working_commerce_price' => [
```

Comment:
You can use the 'template' key to reuse the same template for multiple theme implementations.